src/modules/geolocation/components/GeolocationInput.tsx

```diff
@@ -0,0 +1,105 @@
+import {
```

Comment:
I've been mildly regretting introducing language of "geo" since it's long and complicated compared to simply "location." I just googled to find out if there is actually a semantic difference, and was reminded that "geolocation" refers to getting a device's location (like via satellite or just IP address).

Given that in the future this input will (probably?) also support entering an address and/or dragging and dropping a pin, maybe LocationInput is the more appropriate, as well as simpler, name?

Reply:
I agree on searchability, "location" alone might be challenging. `clientLocation` could be sufficient, but maybe not worth changing

Reply:
I remembered that there is actually somewhere we use the "geolocation" language in a user-facing context, it's in the PIT form. Even if we update that (not in scope for this PR), "geolocation" language is also part of the form definition API, and that for sure doesn't seem worth changing. So I agree that consistently using "geolocation" could help with searchability. I'm good with whatever you decide @gigxz !